rmgpy/tools/canteramodel.py

```diff
@@ -394,6 +448,12 @@ def simulate(self):
                 newkey = get_species_identifier(key)
                 new_mol_frac[newkey] = value
 
+            if self.surface:
+                new_surface_mol_frac = {}
+                for key, value in condition.surface_mol_frac.items():  # this needs to be aspecies object
```

Comment:
Rather than calling the iterating variables key and value, could we pick more informative names like species and mol_fraction?